deployment/desktop/software-appdata.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Adding Applications to GNOME Software
+section: deployment
+subsection: desktop
+description: >
+  How to build and package your GUI desktop applications for easy deployment by your users
+---
+
+# Adding Applications to GNOME Software
+
+The GNOME Software app provides a cross-distribution software center from which users can browse and manage apps for their Linux system.  This guide will help you produce the necessary metadata so your app can be listed in the software center.  Note that this guide covers apps as opposed to other system add-on components such as drivers and codecs.
+
+![GNOME Software](/content/deployment/desktop/steam-gnome-software.png  "Example of Steam listing in GNOME Software")
+
+This guide assumes an application package called Frobnicator, which contains an executable called *frob*.  It also assumes you are running Fedora Workstation with the standard GNOME desktop environment.
+
+## Install helper utilities
+
+Install the following packages on your Fedora system to help assemble
+and validate the metadata for your application:
+
+```console
+$ sudo dnf install desktop-file-utils appstream-util gnome-shell-extension-screenshot-window-sizer
+```
+
+## Create a desktop application file
+
+1. Write a *.desktop* file that follows the [Desktop File
+   Specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#introduction).
+   The required fields are *Name*, *Comment*, *Icon*, *Categories*,
+   *Keywords*, and *Exec*.  Here is an example:
+   
+   ```console
```

Comment:
This is definitely not a `console`. You can leave it empty if you cannot decide.

Reply:
The type is `ini`.